adapters/handlers/grpc/server.go

```diff
@@ -39,6 +39,9 @@ import (
 	v1 "github.com/weaviate/weaviate/adapters/handlers/grpc/v1"
 )
 
+// const maxMsgSize = 104858000 // 10mb, needs to be synchronized with clients
+const maxMsgSize = 999_929400000
```

Comment:
This has been changed upstream. Ideally we don't change this for cuVS as client max message size and server max message size are linked.

Reply:
I did this because with cuVS, building an index all at once is much faster than building parts of it and then rebuilding/extending. I agree that the compatibility issues are more important though